apps/omg/lib/omg/fees.ex

```diff
@@ -34,7 +34,7 @@ defmodule OMG.Fees do
   where fees is itself a map of token to fee spec
   """
   @type full_fee_t() :: %{non_neg_integer() => fee_t()}
-  @type optional_fee_t() :: fee_t() | :no_fees_required
+  @type optional_fee_t() :: fee_t() | :no_fees_required | :no_fees_allowed
```

Comment:
IMHO `no_fees_required` fits better to `merge` txs in the sense that network does not require fees for such txs... <br/>:point_right:  `no_fees_allowed` sounds to me like warning message :)

How about `fees_waived` to diff from fees-free network?